awx_collection/plugins/modules/tower_inventory_refresh.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_inventory_refresh
+author: "FERREIRA Christophe"
+short_description: create, update, or destroy Ansible Tower inventory.
+description:
+    - Refresh all source in inventory
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    name:
+      description:
+        - The name to use for the inventory.
+      required: True
+      type: str
```

Comment:
Since inventory name is not unique, this should take `organization`

Reply:
ok i have add organization to option and put control with get_one